Chapter4/Inheretense/src/Customer.java

```diff
@@ -0,0 +1,10 @@
+
+public class Customer extends Person {
+	
```

Comment:
Please add some comments in the program as below:

/**
 * @author Aamir
 * @since <some date>
 * @description <some description about the class goes in here>
 */

The above comments must go in the beginning of the file (before class declaration)